tests/1_host_configuration.sh

```diff
@@ -71,140 +71,200 @@ fi
 
 # 1.9
 check_1_9="1.9  - Audit Docker files and directories - /var/lib/docker"
-command -v auditctl >/dev/null 2>&1
-if [ $? -eq 0 ]; then
-  auditctl -l | grep /var/lib/docker >/dev/null 2>&1
+directory="/var/lib/docker"
```

Comment:
Docker places files in root "/" of the container. Would this be a safe means of verifying that it has been run in the context of a container. And confirming wether "/var/lib/docker" has been mounted or not?

In the case it has been executed within a container and doesn't have access to "/var/lib/docker"  it can raise a warning (not a crit/error).
In the case it has been executed outside of a container and it has been executed under root then